Makefile

```diff
@@ -6,3 +6,6 @@ linting:
 
 test:
 	PYTHONPATH=${PYTHONPATH}:./app pytest
+
+compose:
+	docker compose up
```

Comment:
Ok. This will run the main docker-compose configuration. Not sure if we will use this in the production. For local compose there probably might be local configuration file.
`docker compose -f local_docker_compose.yml up`